cosima_cookbook/querying.py

```diff
@@ -180,6 +182,11 @@ def _ncfiles_for_variable(expt, variable, session,
             )
         )
 
+    # check whether the results are unique
+    unique_files = set(os.path.basename(f.NCFile.ncfile) for f in ncfiles)
+    if len(unique_files) > 1:
+        warnings.warn("Your query gets a variable from multiple files: {}".format(unique_files))
```

Comment:
Can the warning give the user an indication of how this might be fixed i.e. how to disambiguate the query?

Reply:
Yep, good point!